.circleci/config.yml

```
@@ -41,17 +41,17 @@ jobs:
           name: Check versions
           command: |
             node --version
-            npm --version
+            yarn --version
             docker --version
 
       # Download and cache dependencies
       - restore_cache: *restore_cache
-      - run: *npm_install
+      - run: *yarn_install
       - save_cache: *save_cache
 
       - run:
           name: Build
-          command: npm run build
+          command: yarn run build
```

with yarn you  don't need `run`

```suggestion
          command: yarn build
```